bindings/java/nz/mega/sdk/MegaApiJava.java

```diff
@@ -214,6 +214,23 @@ void runCallback(Runnable runnable) {
     public final static int SEARCH_TARGET_ROOTNODE = MegaApi.SEARCH_TARGET_ROOTNODE;
     public final static int SEARCH_TARGET_ALL = MegaApi.SEARCH_TARGET_ALL;
 
+    public final static int BACKUP_TYPE_CAMERA_UPLOAD = 3;
+
+    public final static int CU_SYNC_STATE_ACTIVE = 1;
+    public final static int CU_SYNC_STATE_FAILED = CU_SYNC_STATE_ACTIVE + 1;
+    public final static int CU_SYNC_STATE_TEMPORARY_DISABLED = CU_SYNC_STATE_FAILED + 1;
+    public final static int CU_SYNC_STATE_DISABLED = CU_SYNC_STATE_TEMPORARY_DISABLED + 2;
+    public final static int CU_SYNC_STATE_PAUSE_UP = CU_SYNC_STATE_DISABLED + 1;
+    public final static int CU_SYNC_STATE_PAUSE_DOWN = CU_SYNC_STATE_PAUSE_UP + 1;
```

Comment:
CU are always uploads. I think this definition doesn't make sense.

Reply:
Yes, that's right.